src/main/java/com/google/sps/servlets/GenerateServlet.java

```diff
@@ -175,8 +176,10 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
   /* Store Scavenger Hunt object in Datastore, return Id of created Hunt. */
   public long writeToDataStore(ScavengerHunt scavHunt) {
     String jsonScavHunt = new Gson().toJson(scavHunt);
+    Text scavHuntText = new Text(jsonScavHunt);
     Entity scavHuntEntity = new Entity(Constants.SCAVENGER_HUNT_ENTITY);
-    scavHuntEntity.setProperty(Constants.SCAVENGER_HUNT_ENTITY, jsonScavHunt);
+    scavHuntEntity.setProperty(Constants.SCAVENGER_HUNT_ENTITY, scavHuntText);
+   // scavHuntEntity.setUnindexedProperty(Constants.SCAVENGER_HUNT_ENTITY, jsonScavHunt);
```

Comment:
Can this be deleted?

Reply:
+1

Reply:
ahhh sorry! yes i will delete the comments